targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs

```diff
@@ -103,6 +103,10 @@ public class PlayFabApiTest
         TitleData resultData = gson.fromJson(testTitleJson, new TypeToken<TitleData>(){}.getType());
         PlayFabSettings.TitleId = resultData.titleId;
         USER_EMAIL = resultData.userEmail;
+        String devSecretKey = env.get("DEVELOPER_SECRET_KEY_JAVA_SDK");
```

Comment:
Was this change intended to be included? I don't see how it's related to the Jenkins script move.

Reply:
Yes, JenkinsSdkSetupScripts contained the title data including the developer secret key. In order to remove the dependency to that repo & not have the secret published in a public repo, the pipelines set it as an env var. Suggestions are welcome

Reply:
Oh, I wasn't aware of that. Your solution is fine.